src/main/java/com/atlassian/bitbucket/jenkins/internal/samples/BitbucketSamples.java

```diff
@@ -0,0 +1,91 @@
+package com.atlassian.bitbucket.jenkins.internal.samples;
+
+import com.atlassian.bitbucket.jenkins.internal.config.BitbucketPluginConfiguration;
+import com.atlassian.bitbucket.jenkins.internal.config.BitbucketServerConfiguration;
+import com.google.inject.Injector;
+import hudson.Extension;
+import jenkins.model.Jenkins;
+import org.jenkinsci.plugins.workflow.cps.GroovySample;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.UUID;
+
+public class BitbucketSamples {
+
+    private BitbucketSamples() {
+        //this class is a holder for samples. As such it should never be instantiated
+    }
+
+    @Extension
+    public static class BitbucketMavenSample implements GroovySample {
+
+        @Override
+        public String name() {
+            return "bbs-maven";
+        }
+
+        @Override
+        public String script() {
```

Comment:
Do we want a test for this? I was uhming and and ahing about it. I _think_ we do? I put the code up for review without a test so let me know what you think

Reply:
I decided to add a test, seems like it is worth it. :)